test/crud-with-options.test.js

```diff
@@ -536,6 +536,60 @@ describe('crud-with-options', function() {
 
   });
 
+  describe('upsertWithWhere', function() {
+    beforeEach(seed);
+
+    it(' rejects upsertWithWhere (options,cb) ', function(done) {
```

Comment:
Don't indent test names, the test runner takes care of making the output pretty.

Bad:

```js
it('  rejects...');
```

Good:

```js
it('rejects ...');
```

Please apply this fix to all test cases below too.